test/e2e/networks/dashcore.toml

```diff
@@ -24,17 +25,20 @@ validator04 = 100
 [validator_update.1010]
 validator01 = 100
 validator02 = 100
+validator03 = 0
```

Comment:
Please double-check that all changes here are correct. Validator updates are tested in rotate.toml.